test/e2e/storage/drivers/in_tree.go

```diff
@@ -1243,7 +1243,7 @@ func InitGcePdDriver() storageframework.TestDriver {
 			},
 			SupportedFsType:      supportedTypes,
 			SupportedMountOption: sets.NewString("debug", "nouid32"),
-			TopologyKeys:         []string{v1.LabelFailureDomainBetaZone},
+			TopologyKeys:         []string{v1.LabelTopologyZone},
```

Comment:
We may have some interesting issues with upgrade tests. We have an upgrade test that runs e2e test version X-1 against a version X cluster, We'll need to figure out how to make that work, but I think it will involve making a change to the test in the release branch. cc @mattcary